.gitlab-ci.yml

```diff
@@ -96,38 +98,63 @@ sniffers:sonarqube:
   - master
   allow_failure: true
 
-prepare:front:
-  cache:
-    key: ${CI_COMMIT_REF_SLUG}-front
-    paths:
-      - ${THEME_PATH}/node_modules/
+
+prepare:back:
   stage: prepare
   script:
-  - make front
+    - docker --version
+    - docker-compose --version
+    - echo "INSTALL_DEV_DEPENDENCIES=TRUE" >> .env.default
+    - make back
+    - docker-compose down # PHP container was required for composer
+  cache:
+    key:
+      files:
+        - composer.json
+        - composer.lock
+    paths:
```

Comment:
I believe cache paths should be the same as artifact paths cause remember that scaffold creates files on web/ folder root level